server/athenian/api/internal/miners/github/deployment.py

```diff
@@ -2022,3 +2024,132 @@ async def load_jira_issues_for_deployments(deployments: pd.DataFrame,
                 prs[pri] = np.sort(np.array(release_keys, dtype="S"))
 
     return issues
+
+
+async def hide_anomal_first_deployments(
```

Comment:
There is no such word "anomal". Replacements: anomalous, outlier. It's better to take the latter to stick to the existing naming.

Reply:
:man_facepalming:  fixed